make_prg/subcommands/from_msa.py

```diff
@@ -52,40 +49,51 @@ def register_parser(subparsers):
         default=MIN_MATCH_LEN,
         help=(
             "Minimum number of consecutive characters which must be identical for a "
-            "match. Default: {}".format(MIN_MATCH_LEN)
+            "match. Default: %(default)s"
         ),
     )
     subparser_msa.add_argument(
-        "-p", "--prefix", dest="output_prefix", action="store", help="Output prefix"
+        "-o",
+        "--output_dir",
+        dest="output_dir",
+        action="store",
+        help="Output directory. Default: location of MSA file",
+    )
+    subparser_msa.add_argument(
+        "-n",
+        "--prg_name",
+        dest="prg_name",
+        action="store",
+        help="Prg file name. Default: MSA file name",
     )
     subparser_msa.add_argument(
         "--no_overwrite",
         dest="no_overwrite",
         action="store_true",
-        help="Do not overwrite pre-existing prg file with same name",
+        help="By default, a prg file which already exists is overwritten and the prg is recomputed. Use this option to keep an existing prg file instead.",
     )
     subparser_msa.set_defaults(func=run)
 
     return subparser_msa
 
 
 def run(options):
-    if options.output_prefix is None:
-        prefix = options.MSA
+    MSA_file = Path(options.MSA).resolve()
+    if not MSA_file.exists():
+        raise ValueError(f"File not found: {options.MSA}")
+    if options.output_dir is None:
+        output_dir = MSA_file.parent
     else:
-        if os.path.isdir(options.output_prefix):
-            prefix = os.path.join(options.output_prefix, os.path.basename(options.MSA))
-        else:
-            prefix = options.output_prefix
-    prefix += ".max_nest%d.min_match%d" % (
-        options.max_nesting,
-        options.min_match_length,
-    )
+        output_dir = Path(options.output_dir).resolve()
```

Comment:
This can all be replaced with `output_dir = Path(options.output_dir)` (if you agree to make the current directory the default)